### DISCUSSION TITLE
Rename to "EMS Reads (Tx)" to "EMS Requests (Tx)" ?

### DISCUSSION THREAD

**Discussion by proddy** [emsesp/EMS-ESP32#443] [Ideas]
From Discord

"What's the relation/difference between EMS Telegrams Received (Rx) and EMS Reads (Tx) ?
My guess:  EMS Reads (Tx) are gateway initiated actions, while EMS Telegrams Received (Rx) are passively received telegrams."

I said "correct. Received Rx is all incoming traffic on the EMS bus. EMS-ESP just listens to these and picks up the ones we know about. EMS Reads (Tx) are explicit calls to the EMS bus requesting data. It uses the Tx part of the circuit logic"

eltoro had a good idea to rename it to "EMS Requests (Tx)" for clarify

?

**Comment by ironirc** [comment]
I propose:
EMS read request
EMS write request

**Comment by mvjt** [comment]
If I read the "specification" the terminology seems to be "Read Command" when requesting a value and "Write Command" when writing a parameter.  Maybe "EMS" is redundant...
